src/color/readableColor.js

```diff
@@ -26,13 +35,30 @@ import getLuminance from './getLuminance'
  *   color: "#fff";
  *   color: "#ff8";
  *   color: "#001";
+ *   color: "#000";
  * }
  */
 
+const defaultLightReturnColor = '#000'
+const defaultDarkReturnColor = '#fff'
+
 export default function readableColor(
   color: string,
-  lightReturnColor: string = '#000',
-  darkReturnColor: string = '#fff',
+  lightReturnColor: string = defaultLightReturnColor,
+  darkReturnColor: string = defaultDarkReturnColor,
+  strict: boolean = false,
 ): string {
-  return getLuminance(color) > 0.179 ? lightReturnColor : darkReturnColor
+  const isLightColor = getLuminance(color) > 0.179
+  const preferredReturnColor = isLightColor ? lightReturnColor : darkReturnColor
+
+  // May return a color that does not meet WCAG AA.
+  // TODO: Make `strict` the default behaviour in the next major release.
+  if (!strict) {
+    return preferredReturnColor
+  }
+
+  if (meetsContrastGuidelines(color, preferredReturnColor).AA) {
+    return preferredReturnColor
+  }
```

Comment:
You should be able to combine these two checks into an `||` check to reduce verbosity.

Reply:
I'll take a look - I was trying to balance verbosity and readability. I'm sure I can clean it up more.